ui/app/templates/components/allocation-row.hbs

```diff
@@ -31,7 +31,7 @@
       role="tooltip"
       aria-label="Allocation was preempted"
     >
-      {{x-icon "boot" class="is-faded"}}
+      <FlightIcon @name="boot" class="is-faded" />
```

Comment:
There is no boot flight icon

Reply:
Take my money 
![image](https://github.com/hashicorp/nomad/assets/174740/9b1a4c11-bf49-4f0c-8c41-c7c09de27796)